commit b344ed7bc3b7e1aeed9c2f8f1ec089117a444366
Author: Theodore Ts'o <tytso@mit.edu>
Date:   Mon Jul 19 02:23:12 2010 -0400

    resize2fs.8: Update man page to indicate ext4 supports on-line resize
    
    Addresses-Debian-Bug: #589345
    
    Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>

Index: e2fsprogs-1.41.12/resize/resize2fs.8.in
===================================================================
--- e2fsprogs-1.41.12.orig/resize/resize2fs.8.in
+++ e2fsprogs-1.41.12/resize/resize2fs.8.in
@@ -31,7 +31,7 @@ enlarge or shrink an unmounted file syst
 If the filesystem is mounted, it can be used to expand the size of the
 mounted filesystem, assuming the kernel supports on-line resizing.  (As
 of this writing, the Linux 2.6 kernel supports on-line resize for
-filesystems mounted using ext3 only.).
+filesystems mounted using ext3 and ext4.).
 .PP
 The 
 .I size
